doc/operators/ledger.rst

```diff
@@ -0,0 +1,56 @@
+Ledger
+======
+
+All network state is contained in the ledger. A single up-to-date copy of the ledger is enough to start a successor
+service if necessary, following the :ref:`operators/recovery:Catastrophic Recovery` procedure.
+
+Each node in a network creates and maintains its own local copy of the ledger. Committed entries are always identical,
+but a node may be more or less up to date, and uncommitted entries may differ.
+
+Location
+--------
+
+Nodes can be configured to store their ledger under a particular directory with the `--ledger-dir` command-line option.
+
+File layout
+-----------
+
+The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-max-bytes`` command line option.
+
+Files containing only committed entries are named ``ledger_$STARTSEQNO-$ENDSEQNO.committed``. These files are closed and immutable,
+it is safe to replicate them to backup storage. They are identical across nodes, provided ``--ledger-chunk-max-bytes`` has been set to the same value.
```

Comment:
I would explicitly state that they cannot be deleted from local storage (for now)

Reply:
Yes, very good point, thank you.